scripts/zones/Pashhow_Marshlands_[S]/mobs/Sugaar.lua

```diff
@@ -0,0 +1,36 @@
+-----------------------------------
+-- Area: Pashhow Marshlands [S] (90)
+--  NM:  Sugaar
+-- !pos -416.496,24.11,-441.589,132
+-----------------------------------
+require("scripts/globals/status");
+require("scripts/globals/magic");
+require("scripts/globals/msg");
+
+function onMobSpawn(mob)
+    mob:setMod(dsp.mod.STORETP, 20);

```

Comment:
No source, I can remove it np if that is preferred.  Sorry I should have put this in the notes but the NM seems to be a larger threat and has minimal online information so I figured it would be accurate to give it some to increase its strength.

Reply:
if its spammy its more likely to be regain than storetp. we'd need to test in retail to determine if it has it (if it still gains tp without being fed any, its got regain for sure).

Reply:
for now remove till we know if it has it.

Reply:
Ok that change should be done assuming I pushed it properly.